test-requirements.txt

```diff
@@ -12,5 +12,7 @@ freezegun
 simplejson
 mock
 Sphinx
-# git+https://github.com/alikins/pyqver.git#egg=pyqver
+yanc
```

Comment:
I would nix this as a test requirement.  It just makes the output colorful and it has some [problems](https://github.com/0compute/yanc/issues/7) (the author claims that issue has been fixed in nose, but I still hit it when I was using yanc..

If a developer wants it, they can always `pip` install it themselves, but it's not really a requirement.  Same with `xtraceback`.  And we'd need to also remove the stuff in the `.noserc`/`setup.cfg` that references these plugins